pygaggle/run/evaluate_kaggle_highlighter.py

```diff
@@ -45,52 +41,43 @@ class KaggleEvaluationOptions(BaseModel):
     split: str
     do_lower_case: bool
     metrics: List[str]
-    model_name: Optional[str]
+    model: Optional[str]
```

Comment:
I prefer the old `model_name` as it avoids confusion with the actual `model` in the code below.

Reply:
We use `model` in our entire codebase so I just changed this to be consistent!